python/ray/_private/node.py

```diff
@@ -1175,38 +1193,18 @@ def _write_cluster_info_to_kv(self):
 
         ray_usage_lib.put_cluster_metadata(self.get_gcs_client())
         # Make sure GCS is up.
-        self.get_gcs_client().internal_kv_put(
-            b"session_name",
-            self._session_name.encode(),
-            True,
-            ray_constants.KV_NAMESPACE_SESSION,
-        )
-        self.get_gcs_client().internal_kv_put(
-            b"session_dir",
-            self._session_dir.encode(),
-            True,
-            ray_constants.KV_NAMESPACE_SESSION,
-        )
-        self.get_gcs_client().internal_kv_put(
-            b"temp_dir",
-            self._temp_dir.encode(),
-            True,
-            ray_constants.KV_NAMESPACE_SESSION,
-        )
+        self._get_or_set_value(b"session_name", self._session_name)
```

Comment:
1. We should update self._session_name variable and  variables of others keys in python , if these keys are persisted.   These variables may be used in the subsequent process.

```
self._session_name
... ohters
```

2. We should print log , if these keys are persisted and resue them.